rulebook.tex

```diff
@@ -1400,45 +1415,53 @@ \subsubsection{Production, Color Complexities, and Additional Bases}
   \label{fig:production-chains}
 \end{figure}
 
-\subsubsection{MPS --- During Production Phase}
+\subsubsection{MPS Interactions}
 \label{sec:production-machines-production}
 In the production phase a production machine performs refinement steps on
 a product such as mounting an additional ring or a cap. Machines
 operate in a transaction style. That is, before using a machine it
-must be prepared for a specific mode. Then the input products can be fed
-into the machine and the refinement step commences. Eventually, after
-the production period is completed, the resulting product is delivered
-on the output lane.
+must be prepared for a specific mode. Most machines take any workpiece
+placed on the input of the machine, perform its operation, and move the
+resulting modified workpiece to the output. Exceptions are \ac{DS}, \ac{BS},
+ and \ac{SS}.
+A \ac{DS} only consumes workpieces, a \ac{BS} can only release (fresh)
+workpieces, whereas a \ac{SS} only does one step (consumption or release)
+at a time.
 
 A light signal mounted near the output lane
 (cf. \refsec{sec:machines}) indicates the state of the machine.
 The light signals are summarized in
 \reftab{tab:machine-lights}.
 
-In the production phase, machines must be prepared to be used by instructing
+Machines must be prepared to be used by instructing
 the refbox. This applies to all machine types.
 If a product from the input is required for the processing step
 (\ac{DS}, \ac{CS}, \ac{RS} and a storage operation at the \ac{SS}),
 then the conveyor belt is started
 after receiving a prepare message. If no workpiece reaches the
 machine operating point (indicated through the middle belt sensor)
-within \SI{45}{\second}, the machine goes to a temporary out-of-order
-state (cf.~\refsec{sec:broken-machine}).
+within \SI{20}{\second}, the machine goes to a temporary out-of-order
```

Comment:
This should be linking to the {Unintended Use of MPS -- } section, as this is exactly what is happening.